docs/days/day-5.md

```diff
@@ -16,7 +16,7 @@ in `templates/job/show.html.twig`:
 - <a class="btn btn-default" href="#">
 + <a class="btn btn-default" href="{{ path('job.list') }}">
       <span class="glyphicon glyphicon-menu-left" aria-hidden="true"></span>
-      Back to list
+      Back to the job list
```

Comment:
I prefer not to change the part with code without any significant cause, because it implies changes in https://github.com/gregurco/jobeet from 5th day to the last... so better sometimes to keep code as it is

Reply:
fair enough